content/strategy-goals/strategy/code-graph/index.md

```diff
@@ -49,3 +49,7 @@ We see Sourcegraph playing an important role in this emerging need for complexit
 If we achieve these, we make it easy for developers to get things done and, ultimately, ship great code.
 
 [1]: https://future.a16z.com/the-case-for-developer-experience/
+
+## Technology and defensibility
+
+On top of building features that are immediately useful to customers, Code graph is investing in key technology areas that will drive Sourcegraph's long term product differentiation and defensibility. Code Intelligence's technology and indexing capabilities, as well as Code Search's compute platform will provide the foundations for delivering a unique, hard to replicate experience across use cases and features, allowing developers to navigate, understand and manipulate their Code Graph.
```

Comment:
Could this be stated simply in one of the principles in the section above? Maybe:

- 🔬 **Innovative**: Investing in underlying search technologies to manipulate the code graph helps us differentiate as well as defend our leading market position.

I like the addition, I'd just like to see it integrated into our existing content. A new section at the bottom could be easy to miss.

On top of building features that are immediately useful to customers, Code graph is investing in key technology areas that will drive Sourcegraph's long term product differentiation and defensibility. Code Intelligence's technology and indexing capabilities, as well as Code Search's compute platform will provide the foundations for delivering a unique, hard to replicate experience across use cases and features, allowing developers to navigate, understand and manipulate their Code Graph.

Reply:
Makes sense, than you! I just committed a proposal.

Reply:
Do we consider here Code Intelligence to fit in the "underlying search technologies"?

Reply:
I'm not sure I understand your question, but my take is that "Code Intelligence the team" is building two things:
- an immediately useful feature called Code Intelligence
- the underlying technology (including indexers) that, as a platform, can be used by multiple product areas and is hard for someone else to replicate.
Is that a mischaracterization?